presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1014,7 +1033,7 @@ public synchronized ConnectorCommitHandle commit()
         try {
             switch (state) {
                 case EMPTY:
-                    return EMPTY_HIVE_COMMIT_HANDLE;
+                    return new HiveCommitHandle(lastDataCommitTimes);
```

Comment:
why did this change?  also, the list is the lastDataCommitTimes for partitions read, but I think here we're trying to write.

Reply:
We already solved the partition creation/alter part. This change is solely for the partition reads.